opba-protocols/protocol-bpmn-shared/src/test/java/de/adorsys/opba/protocol/bpmnshared/config/flowable/LargeJsonCustomSerializerTest.java

```diff
@@ -0,0 +1,45 @@
+package de.adorsys.opba.protocol.bpmnshared.config.flowable;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.Random;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class LargeJsonCustomSerializerTest {
+    private static LargeJsonCustomSerializer serializer;
+
+    @BeforeAll
+    public static void init() {
```

Comment:
Instead of using ObjectMapper use its mock (you are anyway building it yourself):
```
    private static JsonCustomSerializer serializer;
    private static ObjectMapper mapper;

    @BeforeAll
    @SneakyThrows
    public static void init() {
        mapper = mock(ObjectMapper.class);
        when(mapper.writeValueAsString(any())).thenAnswer(inv -> inv.getArgument(0, String.class));
        serializer = new JsonCustomSerializer(null, mapper, Arrays.asList("java.lang.String"), 2048);
    }
```
In this case, test logic will be much cleaner

Reply:
Done for both tests